### PR TITLE
Restructured to orig structure, changed readme doc

### PR DESCRIPTION
**Pull request by ibxibx**

### CODE REVIEW

File: src/components/main-view/main-view.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React, { useState, useEffect } from "react";
-import MovieCard from "../movie-card/movie-card";
-import MovieView from "./movie-view/movie-view";
+import { MovieCard } from "../movie-card/movie-card";
+import { MovieView } from "../movie-view/movie-view";
```
Comment (stratospherique) [inline src/components/main-view/main-view.jsx:3]: The Brackets here will make the import statement incorrect and cause the app's server to fail on compile.
The two files contain default export so you cannot use brackets.

**Review by stratospherique — CHANGES_REQUESTED** [review]
The server is not compiling because of the import statement on movie-card and movie-view files when you try to import components. Kindly delete them.

For example on movie-view.jsx, you can not add instruction like: `import MovieView from "./movie-view";` You're already defining a component named Movie-View, There is no need to import one from outside.

**Review by ibxibx — COMMENTED** [review]
reviewed the changes

File: src/components/main-view/main-view.jsx (hunk 1 above)
Comment (stratospherique) [reply src/components/main-view/main-view.jsx:3]: Here you didn't apply my suggestions about imports. I also provided an article explaining how it works. Please pay close attention

**Comment by stratospherique** [comment]
- The proptypes definition for the movie-view and movie-card is incorrect. The name of the passed prop and its attributes should have the same name in the proptypes definition(same name and its case-sensitive)
  - For example, in main-view, the movie object was defined to contain an attribute called `genre`, but in the movie-card you defined the movie prop to expect a movie object that contains an attribute called `GenreID`(totally different).

**Pull request merged**